### ISSUE TITLE
Add social media accounts for hvplot

### ISSUE THREAD

**Issue by MarcSkovMadsen** [holoviz/holoviz#311]
## Request

Add accounts on Twitter and LinkedIn for hvplot

## Motivation

HoloViz recommends hvPlot as the entry point for HoloViews and the rest of the HoloViz ecosystem. But in fact it is less visible and supported than the rest of the ecosystem. Also on social media.

Without social media accounts tweets and posts on hvplot are different to keep track and history of.

I would like to be able to mention hvPlot in tweets and posts.

**Comment by maximlt** [comment]
I agree that hvPlot should be more present on the social media. I don't know if the right strategy is to have a Twitter account per package. I think right now Panel, HoloViews and DataShader have a Twitter account, and HoloViz also has one.

I will move this issue to the `holoviz` repo as I believe it's a general discussion to have, i.e. how many accounts should there be, who should take care of them, what should they post, etc.

I disagree that hvPlot is less supported than the rest of the ecosystem. That would be GeoViews in my opinion, but it doesn't really matter.

**Comment by jbednar** [comment]
Very good issues to raise, thanks. I think that the Panel, Datashader, and HoloViz accounts each should remain as separate accounts; they have distinct  audiences and very different content on their feeds. 

The question then is the HoloViews account. I don't often want to post specifically about HoloViews, particularly now that we recommend hvPlot as the entry point. Maybe the best solution is to rename the HoloViews account to hvPlot, to emphasize that it's what we want people to focus on?

**Comment by maximlt** [comment]
I'd be happy to tweet about the next hvPlot release if there was an account I could have access to, either a new one dedicated to hvPlot or the old HoloViews after it's renamed, if that happens. I don't have a preference between these two options. Renaming to redirect to another project seems weird, yet this has the advantage not to have to build up a new account from scratch (HoloViews account has about 4k followers, on par with Datashader).
